pipeline/scripts/15-dIEM_violin.R

```diff
@@ -21,12 +26,15 @@ library(gridExtra) # for table top highest/lowest
 #source("/hpc/dbg_mz/production/DIMS/pipeline/scripts/AddOnFunctions/create_violin_plots.R")
 #source("/hpc/dbg_mz/production/DIMS/pipeline/scripts/AddOnFunctions/prepare_alarmvalues.R")
 #temporary:
-source("/hpc/dbg_mz/development/DIMS_Violinplots/pipeline/scripts/AddOnFunctions/check_same_samplename.R")
-source("/hpc/dbg_mz/development/DIMS_Violinplots/pipeline/scripts/AddOnFunctions/prepare_data.R")
-source("/hpc/dbg_mz/development/DIMS_Violinplots/pipeline/scripts/AddOnFunctions/prepare_data_perpage.R")
-source("/hpc/dbg_mz/development/DIMS_Violinplots/pipeline/scripts/AddOnFunctions/prepare_toplist.R")
-source("/hpc/dbg_mz/development/DIMS_Violinplots/pipeline/scripts/AddOnFunctions/create_violin_plots.R")
-source("/hpc/dbg_mz/development/DIMS_Violinplots/pipeline/scripts/AddOnFunctions/prepare_alarmvalues.R")
+source("/hpc/dbg_mz/development/DIMS_output_Helix/pipeline/scripts/AddOnFunctions/check_same_samplename.R")
+source("/hpc/dbg_mz/development/DIMS_output_Helix/pipeline/scripts/AddOnFunctions/prepare_data.R")
+source("/hpc/dbg_mz/development/DIMS_output_Helix/pipeline/scripts/AddOnFunctions/prepare_data_perpage.R")
+source("/hpc/dbg_mz/development/DIMS_output_Helix/pipeline/scripts/AddOnFunctions/prepare_toplist.R")
+source("/hpc/dbg_mz/development/DIMS_output_Helix/pipeline/scripts/AddOnFunctions/create_violin_plots.R")
+source("/hpc/dbg_mz/development/DIMS_output_Helix/pipeline/scripts/AddOnFunctions/prepare_alarmvalues.R")
+source("/hpc/dbg_mz/development/DIMS_output_Helix/pipeline/scripts/AddOnFunctions/output_Helix.R")
+source("/hpc/dbg_mz/development/DIMS_output_Helix/pipeline/scripts/AddOnFunctions/get_patient_data_to_Helix.R")
```

Comment:
I have mentioned this before (to Mia). I think it is best that the development and main branch only contain production ready code. Now you have to remember to update these lines before releasing.

Reply:
Agree, is changed